src/client.rs

```diff
@@ -53,13 +54,13 @@ impl Client {
         let mut client = try_!(new_hyper_client());
         client.set_redirect_policy(::hyper::client::RedirectPolicy::FollowNone);
         Ok(Client {
-            inner: Arc::new(ClientRef {
-                hyper: RwLock::new(client),
-                redirect_policy: Mutex::new(RedirectPolicy::default()),
-                auto_referer: AtomicBool::new(true),
-                auto_ungzip: AtomicBool::new(true),
-            }),
-        })
+               inner: Arc::new(ClientRef {
+                                   hyper: RwLock::new(client),
+                                   redirect_policy: Mutex::new(RedirectPolicy::default()),
+                                   auto_referer: AtomicBool::new(true),
+                                   auto_ungzip: AtomicBool::new(true),
+                               }),
+           })
```

Comment:
I've turned the option on to do this (although note that it separates each call and constructor onto a new line, instead of allowing them to be grouped by pairs.